apps/perf-test-react-components/package.json

```diff
@@ -27,7 +27,7 @@
     "@fluentui/react-spinbutton": "*",
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31258&commitId=909abbca15e5f4f7b76d2733317371fde45803db&env=prod&clientType=FLUENTUI)


  <details><summary><b>TagPicker</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TagPicker.disabled - Dark Mode.chromium.png|1226| Changed|
|TagPicker.expand icon - RTL.chromium.png|246| Changed|
|TagPicker.disabled - RTL.chromium.png|1194| Changed|
|TagPicker.disabled.chromium.png|1185| Changed|
|TagPicker.secondary action - Dark Mode.chromium.png|448| Changed|
|TagPicker.disabled - High Contrast.chromium.png|2229| Changed|
|TagPicker.expand icon - High Contrast.chromium.png|500| Changed|
|TagPicker.expand icon.chromium.png|244| Changed|
|TagPicker.expand icon - Dark Mode.chromium.png|260| Changed|
|TagPicker.secondary action - RTL.chromium.png|319| Changed|
|TagPicker.secondary action - High Contrast.chromium.png|820| Changed|
|TagPicker.secondary action.chromium.png|318| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>